core/net.go

```diff
@@ -96,7 +96,7 @@ func (n *OpenBazaarNode) SendOfflineMessage(p peer.ID, k *libp2p.PubKey, m *pb.M
 	log.Debugf("Sending offline message to: %s, Message Type: %s, PointerID: %s, Location: %s", p.Pretty(), m.MessageType.String(), pointer.Cid.String(), pointer.Value.Addrs[0].String())
 	OfflineMessageWaitGroup.Add(2)
 	go func() {
-		ctx, cancel := context.WithCancel(context.Background())
+		ctx, cancel := context.WithTimeout(context.Background(), n.OfflineMessageFailoverTimeout)
```

Comment:
Since this is in a separate goroutine I don't think we need a timeout here.

Reply:
Are we good on the other ones?

Reply:
I think the other ones are OK. If you remove this one we can merge.